style.css

```diff
@@ -402,3 +430,102 @@ body {
   margin-right: 16px;
   margin-left: 0;
 }
+
+footer {
+  background-color: #6070ff;
+  background-image: url(images/footer.svg);
+  border-top-left-radius: 100px;
+  background-position: right;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 375px;
+  height: 779px;
+}
```

Comment:
- [ ] Great job with your design 👍 . Please, make sure you fix the space around the contact form so it looks as close as possible to the Figma design.

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177427178-3802a045-3c42-483b-bf50-16b5eb900272.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177427226-bf5e6129-002e-414c-aaa5-adbb2cb9464f.png)